api/capi/include/nnstreamer-capi-private.h

```diff
@@ -278,6 +284,31 @@ void ml_tensors_info_copy_from_ml (GstTensorsInfo *gst_info, const ml_tensors_in
  */
 GstCaps * ml_tensors_info_get_caps (const ml_tensors_info_s *info);
 
+/**
+ * @brief Creates a tensor data frame wihout buffer with the given tensors information.
+ * @param[in] info The handle of tensors information for the allocation.
+ * @param[out] data The handle of tensors data.
+ * @return @c 0 on success. Otherwise a negative error value.
```

Comment:
I have a minor question. What is the meaning of @c?

Reply:
display font
```
@a italics
@c typewriter font
```